docs/release-train.md

```diff
@@ -13,7 +13,7 @@ The train should run *often enough* to not slow down development, while also giv
 A release train takes the versions that are currently deployed on one environment and deploys those version to another environment.
```

Comment:
from the commit message:
> However, because of backward compatibility this field is not forced
Why not? This is a backwards incompatibility only within grpc requests, so we have control over everyone who calls them.
We should update all places that call them, and be done with it.